### PR TITLE
[EDU-2633] Modify Services/Support menu structure and adjustments

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Removed service plans page
- Modified links to website pricing page
- Adjusted navigation to remove technical support from professional services
- Made small adjustments to text in all services pages

[EDU-2633]: https://aziontech.atlassian.net/browse/EDU-2633?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by robsongajunior — APPROVED** [review]
lgtm

File: src/content/docs/en/pages/services/integration-services/custom-education-programs.mdx
Comment (ederson-vieira) [inline src/content/docs/en/pages/services/integration-services/custom-education-programs.mdx:1]: Just to check. It was asked to remove the Azion Community reference?

File: src/content/docs/pt-br/pages/servicos/technical-support.mdx [hunk 1]
```diff
@@ -37,12 +35,11 @@ O usuário deve estar previamente autorizado pelo cliente e configurado na ferra
 
 | Plano de Serviço | Canais de Suporte Técnico |
 | ---------------- | ------------------------- |
-| Developer | Acesso ilimitado a documentação, guias, whitepapers e relatórios de bugs. |
 | Business | Suporte 24 horas por dia, 7 dias por semana, via ticket e email. |
-| Enterprise | Suporte 24/7 via ticket, email e telefone, acesso 8/5 a Engenheiros de P&D via ticket e telefone. |
-| Mission Critical | Suporte 24/7 via ticket, email e telefone, acesso 8/5 a Engenheiros de P&D via ticket e telefone. |
+| Enterprise | Suporte 24×7 via ticket, email e telefone, acesso 8/5 a Engenheiros de P&D via ticket e telefone. |
```
Comment (ederson-vieira) [inline src/content/docs/pt-br/pages/servicos/technical-support.mdx:39]: If we changed 24/7 by 24x7, shouldn't we change 8/5 by 8x5?

File: src/content/docs/pt-br/pages/servicos/technical-support.mdx [hunk 2]
```diff
@@ -37,12 +35,11 @@ O usuário deve estar previamente autorizado pelo cliente e configurado na ferra
...
+| Mission Critical | Suporte 24×7 via ticket, email e telefone, acesso 8/5 a Engenheiros de P&D via ticket e telefone. |
```
Comment (ederson-vieira) [inline src/content/docs/pt-br/pages/servicos/technical-support.mdx:40]: Same as above.

File: src/content/docs/en/pages/services/integration-services/best-practices-review.mdx [hunk 3]
```diff
@@ -6,16 +6,23 @@ namespace: documentation_services_best_practices_review
 permalink: /documentation/services/best-practices-review/
 ---
 
-Azion's **Best Practices Review (BPR)** offers a thorough evaluation of your infrastructure and provides expert recommendations to optimize performance, security, and efficiency. Azion's BPR enhances your operations and ensures industry-leading standards for your online presence.
+Azion's **Best Practices Review (BPR)** offers a thorough evaluation of the infrastructure supported by Azion and provides expert recommendations to optimize performance, security, and efficiency. Azion's BPR enhances your operations and ensures industry-leading standards for your online presence.
+
+At Azion, we conduct an in-depth analysis of your workload to:
```
Comment (hannah-sk) [inline src/content/docs/en/pages/services/integration-services/best-practices-review.mdx:11]:
```suggestion
At Azion, an in-depth analysis of your workload is conducted to:
```

File: src/content/docs/en/pages/services/integration-services/best-practices-review.mdx [hunk 4]
```diff
@@ -6,16 +6,23 @@ namespace: documentation_services_best_practices_review
...
+
+- improve application performance and security.
+- assess the appropriate security measures.
+- support cost optimization.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/services/integration-services/best-practices-review.mdx:15]:
```suggestion
- Improve application performance and security.
- Assess the appropriate security measures.
- Support cost optimization.
```

File: src/content/docs/en/pages/services/integration-services/custom-education-programs.mdx [hunk 5]
```diff
@@ -8,8 +8,12 @@ permalink: /documentation/services/custom-education-programs/
...
+For more detailed information, see [Azion Service Plans](https://www.azion.com/en/pricing/#plans).
 
-You can also join the [Azion Community](https://discord.com/invite/Yp9N7RMVZy) on Discord and integrate with a community of developers. 
+In addition to the Custom Education Programs, Azion provides full access to comprehensive documentation through:
+
+- [Azion Docs](/en/documentation/). The documentation is a valuable resource that provides all the information you need to build serverless applications at the edge. You can explore and utilize Azion Docs to build, secure, deploy, and observe powerful applications, enabling you to maximize the potential of Azion's services.
+
+- [Azion Learn](https://learn.azion.com/). In this channel, you have access to  custom learning resources to accomplish a specific project and learn more about the Azion fundamentals.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/services/integration-services/custom-education-programs.mdx:19]:
```suggestion
- [Azion Learn](https://learn.azion.com/). In this channel, you have access to custom learning resources to accomplish a specific project and learn more about the Azion fundamentals.
```

File: src/content/docs/en/pages/services/integration-services/integration-services.mdx [hunk 6]
```diff
@@ -6,35 +6,38 @@ namespace: documentation_services_integration_services
 permalink: /documentation/services/solutions-lab/integration-services/
 ---
 
-**Integration Services** is the fastest and safest way to migrate your content, web applications, and APIs to Azion, no matter how complex and resource-intensive your infrastructure is. The Integration Services team will activate the Azion products and services you've contracted and provide the required tools, documentation, and training for your success using the Azion Platform.
+**Integration Services** is the fastest and safest way to migrate your content, web applications, and APIs to Azion, no matter how complex and resource-intensive your infrastructure is. Our team will provide the required documentation for your success using the Azion Platform.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/services/integration-services/integration-services.mdx:9]:
```suggestion
**Integration Services** is the fastest and safest way to migrate your content, web applications, and APIs to Azion, no matter how complex and resource-intensive your infrastructure is. The team will provide the required documentation for your success using the Azion Platform.
```

File: src/content/docs/en/pages/services/support/technical-support.mdx [hunk 7]
```diff
@@ -8,17 +8,15 @@ permalink: /documentation/services/support/
 
 At Azion, you're served by a team of engineers who are specialized in performance, security, high availability, and scalability of content, web applications, and APIs, especially in mission-critical environments.
 
-Whether you're new or you're increasing the usage of products, Azion wants to provide the right resources to support your success. **Customer Support** offers the expertise to help you do amazing things with Azion.
-
-## Technical Support
+Whether you're new or you're increasing the usage of products, Azion wants to provide the right resources to support your success. Azion offers a unique combination of tools and expertise to help you achieve amazing things.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/services/support/technical-support.mdx:11]: do we need this sentence? very mkt

File: src/content/docs/pt-br/pages/servicos/best-practices-review.mdx [hunk 8]
```diff
@@ -6,16 +6,23 @@ namespace: documentation_services_best_practices_review
 permalink: /documentacao/servicos/best-practices-review/
 ---
 
-O **Best Practices Review (BPR)** da Azion oferece uma avaliação completa de sua infraestrutura e fornece recomendações de especialistas para otimizar o desempenho, a segurança e a eficiência. O BPR da Azion aprimora suas operações e garante padrões comparáveis a líderes do setor para sua presença online.
+O **Best Practices Review (BPR)** da Azion oferece uma avaliação completa da infraestrutura à qual a Azion dará suporte e fornece recomendações de especialistas para otimizar o desempenho, a segurança e a eficiência. O BPR da Azion aprimora suas operações e garante padrões comparáveis a líderes do setor para sua presença online.
+
+Na Azion, realizamos uma análise profunda do seu *workload* para:
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/servicos/best-practices-review.mdx:11]:
```suggestion
Na Azion, é realizada uma análise profunda do seu *workload* para:
```

File: src/content/docs/pt-br/pages/servicos/best-practices-review.mdx [hunk 9]
```diff
@@ -6,16 +6,23 @@ namespace: documentation_services_best_practices_review
...
+
+- melhorar o desempenho e a segurança das aplicações.
+- avaliar as medidas de segurança apropriadas.
+- apoiar a otimização de custos.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/servicos/best-practices-review.mdx:15]:
```suggestion
- Melhorar o desempenho e a segurança das aplicações.
- Avaliar as medidas de segurança apropriadas.
- Apoiar a otimização de custos.
```

File: src/content/docs/pt-br/pages/servicos/integrations.mdx [hunk 10]
```diff
@@ -6,36 +6,39 @@ namespace: documentation_services_integration_services
 permalink: /documentacao/servicos/solutions-lab/integration-services/
 ---
 
-**Integration Services** é a forma mais rápida e segura para migrar seus conteúdos, aplicações web e APIs para a Azion, não importando a complexidade e o volume de recursos que você utiliza. O time de Integration Services irá ativar os serviços contratados por você e prover as ferramentas, documentações e treinamentos necessários para seu sucesso utilizando a plataforma da Azion.
+**Integration Services** é a forma mais rápida e segura para migrar seus conteúdos, aplicações web e APIs para a Azion, não importando a complexidade e o volume de recursos que você utiliza. Nosso time irá prover as documentações necessárias para seu sucesso utilizando a plataforma da Azion.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/servicos/integrations.mdx:9]:
```suggestion
**Integration Services** é a forma mais rápida e segura para migrar seus conteúdos, aplicações web e APIs para a Azion, não importando a complexidade e o volume de recursos que você utiliza. O time irá prover as documentações necessárias para seu sucesso utilizando a plataforma da Azion.
```

File: src/content/docs/pt-br/pages/servicos/technical-support.mdx [hunk 11]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Suporte ao Cliente
+title: Technical Support
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/servicos/technical-support.mdx:2]:
```suggestion
title: Suporte Técnico
```

File: src/i18n/en/nav.ts [hunk 12]
```diff
@@ -66,21 +66,20 @@ export default [
...
-		{ text: 'Custom Education Programs', header: true, anchor: true, type: 'learn', slug: '/documentation/services/custom-education-programs/', key: 'professionalServices/customEducationPrograms' },
-		{ text: 'Security Response Team', header: true, anchor: true, type: 'learn', slug: '/documentation/services/security-response-team/', key: 'professionalServices/SRT' },
-		{ text: 'Technical Support', header: true, anchor: true, type: 'learn', slug: '/documentation/services/support/', key: 'professionalServices/technicalSupport' },
-		{ text: 'Business Events Support', header: true, anchor: true, type: 'learn', slug: '/documentation/services/business-events-support/', key: 'professionalServices/businessEventsSupport' },
-		{ text: 'Slack Channel', header: true, anchor: true, type: 'learn', slug: '/documentation/services/slack-channel/', key: 'professionalServices/SlackChannel' },
-	]
-},
+	{ text: 'Marketplace', header: true, anchor: true, type: 'learn', slug: 'documentation/products/marketplace', key: 'marketplace' },
```
Comment (hannah-sk) [inline src/i18n/en/nav.ts:82]:
```suggestion
```

File: src/i18n/en/nav.ts [hunk 13]
```diff
@@ -66,21 +66,20 @@ export default [
 	
 	] },
 
-	{ text: 'Marketplace', header: true, anchor: true, type: 'learn', slug: 'documentation/products/marketplace', key: 'marketplace' },
+	{ text: 'Technical Support', header: true, anchor: true, type: 'learn', slug: '/documentation/services/support/', key: 'technicalSupport' },
```
Comment (hannah-sk) [inline src/i18n/en/nav.ts:69]:
```suggestion
	{ text: 'Marketplace', header: true, anchor: true, type: 'learn', slug: 'documentation/products/marketplace', key: 'marketplace' },
	
	{ text: 'Technical Support', header: true, anchor: true, type: 'learn', slug: '/documentation/services/support/', key: 'technicalSupport' },
```

File: src/content/docs/en/pages/services/integration-services/integration-services.mdx [hunk 14]
```diff
@@ -6,35 +6,38 @@ namespace: documentation_services_integration_services
...
 1. Activation of the Azion services contracted as described in the *Service Order*.
 2. Gathering of technical and business requirements with the customer.
 3. Real-Time Manager (RTM) usage training.
 4. Configuration of Azion services for the customer.
-5. Review and optimization of service configurations according to customer demands.
+5. Review and optimization of service configurations according to customer demands during the launch.
 
 The Azion specialist will follow an integration process consisting of the steps below:
 
+import TableContainer from "~/components/Table/TableContainer.astro"
```
Comment (peeetry) [inline src/content/docs/en/pages/services/integration-services/integration-services.mdx:29]: Do you know if there is standard for imports declaration? Like, the import should be in the beginning of the page, or right before usage?

**Review by hannah-sk — COMMENTED** [review]
left a few corrections

File: src/content/docs/en/pages/services/integration-services/integration-services.mdx [hunk 15]
```diff
@@ -6,35 +6,38 @@ namespace: documentation_services_integration_services
...
+
+<TableContainer>
```
Comment (peeetry) [inline src/content/docs/en/pages/services/integration-services/integration-services.mdx:31]: Genuine question. If tables and simple <p> already work, what is the point of `<TableContainer>`?

File: src/content/docs/pt-br/pages/servicos/best-practices-review.mdx (hunk 9 above)
Comment (peeetry) [inline src/content/docs/pt-br/pages/servicos/best-practices-review.mdx:15]:
```suggestion
- Melhorar o desempenho e a segurança das aplicações.
- AValiar as medidas de segurança apropriadas.
- Apoiar a otimização de custos.
```

**Review by ghost — APPROVED** [review]

File: src/content/docs/en/pages/services/integration-services/integration-services.mdx (hunk 14 above)
Comment (bru-andrade) [reply src/content/docs/en/pages/services/integration-services/integration-services.mdx:29]: I've seen we use it both ways 😅 as long as it's not after, it should compile just fine

File: src/content/docs/en/pages/services/integration-services/integration-services.mdx (hunk 15 above)
Comment (bru-andrade) [reply src/content/docs/en/pages/services/integration-services/integration-services.mdx:31]: It makes it so the first column is not as squished, like in the old variables table in Rules Engine.

File: src/content/docs/en/pages/services/integration-services/custom-education-programs.mdx
Comment (bru-andrade) [reply src/content/docs/en/pages/services/integration-services/custom-education-programs.mdx:1]: I took a note from the meeting to move the Azion Community link to the Technical Support page. We could add it back here again if needed.

File: src/content/docs/pt-br/pages/servicos/technical-support.mdx (hunk 1 above)
Comment (bru-andrade) [reply src/content/docs/pt-br/pages/servicos/technical-support.mdx]: Done!

File: src/content/docs/en/pages/services/integration-services/custom-education-programs.mdx
Comment (ederson-vieira) [reply src/content/docs/en/pages/services/integration-services/custom-education-programs.mdx:1]: There is no need to add right now. If you took this note, it's ok.

As you said, we can add it later, if needed. Thanks!

**Review by ederson-vieira — APPROVED** [review]

**Pull request merged**